aw-server/src/config.rs

```diff
@@ -116,3 +118,21 @@ pub fn create_config(testing: bool) -> AWConfig {
 
     aw_config
 }
+
+/// Retrieves the device ID, if none exists it generates one (using UUID v4)
+pub fn get_device_id() -> String {
+    // TODO: Use seperate device_id for testing mode
+    // TODO: Cache to avoid retrieving on every /info call
+    // TODO: How should these unwraps be removed?
```

Comment:
It should only be read once, and that's before the server starts.
Then it should simply be passed around like the config.